cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -100,13 +99,10 @@ func absoluteKubeletRunDirectory() (string, error) {
 		klog.Warningf("[reset] Failed to evaluate the %q directory. Skipping its unmount and cleanup: %v\n", kubeadmconstants.KubeletRunDirectory, err)
 		return "", err
 	}
-
-	// Only unmount mount points which start with "/var/lib/kubelet" or absolute path of symbolic link, and avoid using empty absoluteKubeletRunDirectory
-	umountDirsCmd := fmt.Sprintf("awk '$2 ~ path {print $2}' path=%s/ /proc/mounts | xargs -r umount", absoluteKubeletRunDirectory)
-	klog.V(1).Infof("[reset] Executing command %q", umountDirsCmd)
-	umountOutputBytes, err := exec.Command("sh", "-c", umountDirsCmd).Output()
+	err = unmountKubeletDirectory()
```

Comment:
A thing, that I've missed on the first run here is, that we need `absoluteKubeletRunDirectory` to be passed to `unmountKubeletDirectory` (instead of using a const there).

Reply:
+1